helm/alfresco-content-services/charts/activemq/templates/deployment-activemq.yaml

```diff
@@ -58,12 +58,13 @@ spec:
           periodSeconds: {{ .Values.livenessProbe.periodSeconds }}
           failureThreshold: {{ .Values.livenessProbe.failureThreshold }}
           timeoutSeconds: {{ .Values.livenessProbe.timeoutSeconds }}
+        resources:
+{{ toYaml .Values.resources | indent 12 }}
+       {{- if .Values.persistence.dynamicProvisioning }}
```

Comment:
I would argue we would need volume to be mounted even if not using dynamic provisioning